packages/react-native/src/primitives/Icon/Icon.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useMemo } from 'react';
+import { Animated, Image } from 'react-native';
+
+import { getStyles } from './styles';
+import { IconProps } from './types';
+
+export default function Icon({
+  color,
+  animated = false,
```

Comment:
Since an `undefined` value for `animated` will evaluate to falsy, do we need the default value here?